lib/pry/pry_class.rb

```diff
@@ -1,7 +1,17 @@
 require 'pry/config'
 class Pry
 
-  HOME_RC_FILE = ENV["PRYRC"] || "~/.pryrc"
+  HOME_RC_FILE = proc do
```

Comment:
@kyrylo I think I can wrap it in a method in this pull request.

I agree that it's a job for a separate pull request. In fact, I'm not even confident to write test for this; I have no idea how to test file operations reliably. [Usual `StringIO` trick](https://stackoverflow.com/questions/4070422/rspec-how-to-test-file-operations-and-file-content/4070479#4070479) doesn't seem to work here. Injection into `File`? Fake file system?